nixos/platform/ipmi.nix

```diff
@@ -86,6 +86,7 @@ in {
         sleep 1
         ipmitool sol set volatile-bit-rate 115.2 1
         sleep 1
+        ipmitool user set name 2 ADMIN
```

Comment:
Is the administrative user always user number 2? (Do we have a reference for this?)